react-native/ios/Keybase/KeyListener.m

```diff
@@ -0,0 +1,46 @@
+//
+//  KeyListener.m
+//  Keybase
+//
+//  Created by Chris Nojima on 9/28/16.
+//  Copyright © 2016 Keybase. All rights reserved.
+//
+
+#import "KeyListener.h"
+#import "RCTEventDispatcher.h"
+
+@implementation KeyListener 
+
+- (BOOL)canBecomeFirstResponder {
+  return YES;
+}
+
+- (NSArray *)keyCommands {
+  return @[
+           [UIKeyCommand keyCommandWithInput:@"[" modifierFlags:UIKeyModifierCommand action:@selector(goBackInTime:)],
+           [UIKeyCommand keyCommandWithInput:@"]" modifierFlags:UIKeyModifierCommand action:@selector(goForwardInTime:)],
+           [UIKeyCommand keyCommandWithInput:@"s" modifierFlags:UIKeyModifierCommand|UIKeyModifierShift action:@selector(saveState:)],
+           [UIKeyCommand keyCommandWithInput:@"c" modifierFlags:UIKeyModifierCommand|UIKeyModifierShift action:@selector(clearState:)]
+           ];
+}
+
+- (void)goBackInTime:(UIKeyCommand *)sender {
+  [self.bridge.eventDispatcher sendAppEventWithName:@"backInTime"
```

Comment:
I see references to these events in `shared/index.native.js`. Can they be removed now?